### PR TITLE
#549 Provide help for managing the API token

### PR DESCRIPTION
**Pull request by johannes-b**
Explanation of how to retrieve, create, re-create the API token. (in both, the official 0.6.2 docu and develop folder)

### CODE REVIEW

File: content/docs/0.6.0/reference/api_token/index.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    </details>
+    </p>
+    </details>
+
+## Create API Token
+
+* To create an API token, execute:
+  
+    ```console
+    KEPTN_API_TOKEN=$(head -n 16 /dev/urandom | base64)
```
Comment (agrimmer) [inline content/docs/0.6.0/reference/api_token/index.md:90]: Is this command executable in Windows?

File: content/docs/0.6.0/reference/api_token/index.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
+---
+title: API Token
+description: Manage the API token of a Keptn installation.
+weight: 12
+keywords: [api, setup]
+---
+
+In this section, the functionality and commands of the Keptn REST API are described.
```
[the remark this answers is not in the document]
Comment (johannes-b) [reply content/docs/0.6.0/reference/api_token/index.md:8]: Thx, fixed it.

File: content/docs/0.6.0/reference/api_token/index.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+## Retrieve API Token
+
+* To retrieve the API token of a running Keptn installation, execute: 
+
+    ```console
+    kubectl get secret keptn-api-token -n keptn -ojsonpath='{.data.keptn-api-token}'
+    ``` 
+
+* To decode the retrieved API token, use means offered by the operating system:
```
[the remark this answers is not in the document]
Comment (johannes-b) [reply content/docs/0.6.0/reference/api_token/index.md:18]: Removed the part of setting environment variables.

File: content/docs/0.6.0/reference/api_token/index.md (hunk 1 above)
Comment (johannes-b) [reply content/docs/0.6.0/reference/api_token/index.md:90]: No, let me add the Windows-way of creating the token.

Comment (johannes-b) [reply content/docs/0.6.0/reference/api_token/index.md]: I have now provided the instructions for Windows. May I ask for a second review.

**Pull request merged**